sdk/python/tests/performance_tests.py

```diff
@@ -205,91 +224,111 @@ def get_client() -> TektonClient:
     return client
 
 
-def load_pipeline_functions() -> [(Callable, str)]:
```

Comment:
the diff shown from here on down is a little misleading, GH tries to align lines of code that are not related